src/ontology/efo-edit.owl

```diff
@@ -203006,6 +203007,22 @@ AnnotationAssertion(oboInOwl:hasExactSynonym efo:EFO_0011018 "BDNF")
 AnnotationAssertion(rdfs:label efo:EFO_0011018 "brain-derived neurotrophic factor measurement"@en)
 SubClassOf(efo:EFO_0011018 efo:EFO_0004747)
 
+# Class: efo:EFO_0011023 (concussion)
+
+AnnotationAssertion(Annotation(efo:definition_citation "https://en.wikipedia.org/wiki/Concussion") obo:IAO_0000115 efo:EFO_0011023 "A concussion, also known as a mild traumatic brain injury (mTBI), is a head injury that temporarily affects brain functioning. It is usually caused by a bump, blow or jolt to the head.")
+AnnotationAssertion(obo:IAO_0000117 efo:EFO_0011023 "Elizabeth Langley")
+AnnotationAssertion(efo:gwas_trait efo:EFO_0011023 "true"^^xsd:boolean)
+AnnotationAssertion(oboInOwl:hasDbXref efo:EFO_0011023 "PMID:33017352")
+AnnotationAssertion(oboInOwl:hasDbXref efo:EFO_0011023 "http://purl.obolibrary.org/obo/NCIT_C50502")
```

Comment:
Should be:
```
AnnotationAssertion(oboInOwl:hasDbXref efo:EFO_0011023 "NCIt:C50502")
```

to be consistent with other NCIT xrefs (note the lower case t in NCIt as well)